src/utils/novels.js

```diff
@@ -0,0 +1,11 @@
+import fetchApiData from '../actions/novels'
+
+export const fetchData = async () => {
+  const data = await fetchApiData()
+
+  return data
+}
+
```

Comment:
I did not get a search list on my end either....although it may have just been denying me access to the database.  The code looks right so I didn't dig too deep.